tests/framework/db/ActiveQueryTest.php

```diff
@@ -129,6 +129,24 @@ public function testInnerJoinWith()
         ], $result->joinWith);
     }
 
+    /**
+     * @todo: tests for the regex inside getQueryTableName
+     */
+    public function testGetQueryTableName_from_not_set()
```

Comment:
for normal framework code, yes. camelCase will do. 
But not if you are testing:
* 1 method:
* on multiple criteria
* in multiple test methods (to keep the smallest contained environment + to have an instant understanding if a test fails)